.github/workflows/sphinx.yml

```diff
@@ -9,13 +9,14 @@ jobs:
     runs-on: ubuntu-latest
     steps:
 
-      - name: Set up Python 3.9
+      - name: Set up Python 3.1
```

Comment:
Python 3.1 might be a little bit outdated...

Reply:
Oops!